pandas/core/arrays/base.py

```diff
@@ -580,6 +580,33 @@ def factorize(self, na_sentinel=-1):
         uniques = self._from_factorized(uniques, self)
         return labels, uniques
 
+    def repeat(self, repeats, axis=None):
+        """
+        Repeat elements of an array.
+
```

Comment:
Need a versionadded. Will do that on merge maybe, unless there are other comments on the implementation.